src/test/common/protocol.test.ts

```diff
@@ -10,7 +10,7 @@ import { Protocol, ProtocolType } from '../../common/protocol';
 const SSH_CONFIG_WITH_HOST_ALIASES = `
 Host gh
   User git
-  HostName github.com
+  Hostname github.com
```

Comment:
Instead of changing this, could you add an additional test duplicating the last test here but using a config with `Hostname` instead of `HostName` so that we test both?

Reply:
Thank you for the feedback Rachel I have implemented your suggestion of a second test.